### ISSUE TITLE
Accessibility for Status View Glyph Color

### ISSUE THREAD

**Issue by jenmoran** [cerner/terra-core#2222]
# Bug Report

## Steps to Reproduce
1. Go to https://webaim.org/resources/contrastchecker/
2. Enter #EDEEEE as the foreground color. (used CCA color picker to find hex color)
3. Enter #FFFFFF as the background color.

## Additional Context / Screenshots
 I also used the Colour Contrast Analyser (CCA) (https://developer.paciellogroup.com/resources/contrastanalyser/) to select the color from the Terra UI status view page of the glyph which rendered.  
<img width="1094" alt="screen shot 2019-02-11 at 9 06 39 am" src="https://user-images.githubusercontent.com/47461485/52572904-e9d09600-2de6-11e9-83eb-6cc307278df3.png">

<img width="479" alt="screen shot 2019-02-11 at 9 43 57 am" src="https://user-images.githubusercontent.com/47461485/52572983-171d4400-2de7-11e9-8d6b-e86cd37ef247.png">

<img width="1421" alt="screen shot 2019-02-11 at 10 29 39 am" src="https://user-images.githubusercontent.com/47461485/52573369-002b2180-2de8-11e9-95ac-889ed5162a03.png">

## Expected Behavior
Color contrast to pass.

## Possible Solution
Selecting a color that passes accessibility standards.

## Environment

* Component Name and Version: Terra Status View v3.4.0
* Browser Name and Version: Safari
* Node/npm Version: [e.g. Node 8/npm 5]
* Webpack Version:
* Operating System and version (desktop or mobile):

**Comment by bjankord** [comment]
@jenmoran It looks like the color you've sampled is for the icon, #edeeee. However, the errors in the screenshots look like they are for text contrast ratios. Could you provide some more guidance on the issue you are seeing here.

**Comment by scottwilmarth** [comment]
@bjankord I advised @jenmoran regarding the status icons after this issue was logged. When the icon is displayed with the text, the icon does not need to meet contrast ratio as it is just supplementing the text that is accompanying the icon. If the icon were to be the only thing displayed (no text displayed with it) to convey the meaning, the icon would have to meet 3:1 contrast ratio. You should be fine to close this issue out.
